frb_codegen/src/library/commands/cargo_expand.rs

```diff
@@ -111,7 +116,12 @@ fn run_cargo_expand(
 
     let mut stdout_lines = stdout.lines();
     stdout_lines.next();
-    let ans = stdout_lines.join("\n").replace("/// frb_marker: ", "");
+    let ans = stdout_lines.join("\n");
+    static PATTERN: OnceLock<Regex> = OnceLock::new();
+    let pattern = PATTERN.get_or_init(|| {
```

Comment:
hmm, not very sure, this is `cargo_expand` function which does not know what is a frb attribute, maybe we should not do replacement at this stage, but do it at other places

Reply:
We could refactor the post-processing out into another function, but this is the earliest we can do it and doesn't leave doc-comments hanging around for too long. And while cargo-expand doesn't know about `#[frb]`, rustc does because of the --config passed above.

Reply:
I see... Then what about this: Extract frb_expand (the "rustc does know it") *and* the post-processing, into a simple wrapper function. Say, `fn cargo_expand_2()` (bad name, just for demonstration), s.t. the inner cargo_expand dnk frb, only the wrapper knows